pwa/src/components/sources/sourceForm.tsx

```diff
@@ -318,7 +319,7 @@ export const SourceForm: React.FC<SourceFormProps> = ({ sourceId }) => {
                       </div>
                       <div className="col-6">
                         <GenericInputComponent
-                          type={"text"}
+                          type={"password"}
```

Comment:
When you change an `input` to `type=password` you cannot see what you're typing anymore. Since this is not described in the ticket, we will have to verify this.

To do: check with @rubenvdlinde if changing this input to `type=password` is wanted.